lib/rouge/lexers/hcl.rb

```diff
@@ -123,6 +124,7 @@ def self.builtins
       state :array do
         mixin :comments_and_whitespace
 
+        rule %r/[.,()\\\/*]/, Punctuation
         rule %r/,/, Punctuation
```

Comment:
I guess the regex on Line 127 already includes `,`. Hence we can drop Line 128. WDYT?

```suggestion
        rule %r/[.,()\\\/*]/, Punctuation
```

Reply:
Yes agreed. I tested without line 128 and the sample hcl renders correctly still.